test/numerical/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ set(TEST_LINK_LIBS rapidcheck CompilerUtils ExecutionSession)
 
 add_numerical_unittest(TestConv
   TestConv.cpp
-  LINK_LIBS PRIVATE ${TEST_LINK_LIBS}
+  LINK_LIBS PRIVATE ${TEST_LINK_LIBS} ModelLib
```

Comment:
Since this line is getting long, I'd put things on a new line

Reply:
Also, if ModelLib will be used by all of the tests, you can add it to TEST_LINK_LIBS instead

Reply:
sure